### DISCUSSION TITLE
Labels for LCD display

### DISCUSSION THREAD

**Discussion by rooftopsolarsa** [ArminJo/JK-BMSToPylontechCAN#24] [Q&A] [answered]
Great work, setup this weekend and everything works great on my Sunsynk  5K inverter :)
On the LCD display 1st page at the bottom is a voltage, what is that voltage?
Kind Regards
Morne

![BigInfoPage-labels](https://github.com/ArminJo/JK-BMSToPylontechCAN/assets/153637794/b8563431-aee8-4bf9-b5e5-210b81d7fc05)

**Comment by rooftopsolarsa** [comment]
This is the description you have.
Big Info page **with difference voltage between actual and 100%** voltage and display of "C"harging "D"ischarging and "B"alancing active flags.

**Reply by ArminJo** [reply, marked as answer]
The voltage difference to maximum voltage allowed,

**Reply by rooftopsolarsa** [reply]
Perfect thanks, Now I understand.
